resources/assets/js/app/models/VoteAIModel.js

```diff
@@ -2,5 +2,22 @@ var BaseModel = require('../instances/Model');
 var _ = require('underscore');
 
 module.exports = BaseModel.extend({
-    urlRoot: '/voteitems'
-});
\ No newline at end of file
+    urlRoot: '/voteitems',
+    initialize: function (options) {
+        if (options)
+            if (options.parentUrl) {
+                this.parentUrl = options.parentUrl;
```

Comment:
implement this logic in BaseModel, we have a lot of same initializers accross app